python/tk_3dsmaxplus/maxscript.py

```diff
@@ -51,6 +51,13 @@ def create_menu(menu_name, menu_var):
         '''.format(menu_var=menu_var, menu_name=menu_name))
 
     @staticmethod
+    def destroy_menu(menu_name):
+        MaxPlus.Core.EvalMAXScript('''
+            menu = menuMan.findMenu "{menu_name}"
+            if menu != undefined then menuMan.unregisterMenu menu
```

Comment:
why are we using ` MaxPlus.MenuManager.UnregisterMenu(MENU_LABEL)` in one place and maxscript in another? Could we use python in all places? if not, please add a comment explaining a bit!